stdlib/public/Concurrency/AsyncPrefixSequence.swift

```diff
@@ -0,0 +1,69 @@
+////===----------------------------------------------------------------------===//
+////
+//// This source file is part of the Swift.org open source project
+////
+//// Copyright (c) 2021 Apple Inc. and the Swift project authors
+//// Licensed under Apache License v2.0 with Runtime Library Exception
+////
+//// See https://swift.org/LICENSE.txt for license information
+//// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+////
+////===----------------------------------------------------------------------===//
+
+import Swift
+
+extension AsyncSequence {
+  @inlinable
+  public __consuming func prefix(
+    _ maxLength: Int
```

Comment:
We use `k` for this argument on `dropFirst`, and `limit` on the drop first type. Is there some more standard terminology we could apply across all of these?